geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/AbstractHitsMissesIntegrationTest.java

```diff
@@ -252,6 +252,11 @@ public void testZIncrBy() {
     runCommandAndAssertNoStatUpdates("key", (k, m) -> jedis.zincrby(k, 100.0, m));
   }
 
+  @Test
+  public void testZPopMin() {
```

Comment:
Could this be moved to below `testZPopMax()` to maintain alphabetical ordering?

Reply:
Done